sake.config.js

```diff
@@ -13,4 +13,7 @@ module.exports = {
 	framework: 'v5',
 	deployAssets: false,
 	autoload: true,
+	paths: {
+		js: false
```

Comment:
Would be good to document the intention here, it's js file so you can use comments :)

Reply:
Added in 02d753af5c1c0288877ffd87514e279eb17ca4d6